Add support for bash shell.

uncleBlobby opened the pull request:
Add basic command parser for bash history file

Add switch in main.go that checks for the shell type set in config and runs shell-specific parser for each case.

uncleBlobby committed:
Add bash history parser

Add switch statement in main.go to control which parser executes depending on which parser is found during the config

quentinlintz commented on history/parser.go:
```diff
@@ -21,6 +21,8 @@ func PrintTopCommands(cfg config.Config, p Parser) {
 		log.Fatalf("Error parsing history: %v", err)
 	}
 
+	log.Printf("commands in PrintTopCommands: %#v", commands)
+
```
Could we remove this log statement? When I create a release it looks like Printf logs end up in the binary.

uncleBlobby replied:
My bad! I thought I had removed all my debugging logs, but I guess I missed a couple.

I'll remove those and add that test case, then I'll circle back to you. 😁

quentinlintz commented on history/bash_parser.go:
```diff
@@ -0,0 +1,43 @@
+		commands = append(commands, models.Command{Name: name, Count: count})
+	}
+
+	log.Printf("commands: %#v", commands)
```
This could be removed too. Maybe a future feature could be build tags to conditionally display log statements?

quentinlintz reviewed (changes requested):
This is FANTASTIC, thank you!

Could you add a new `history/testdata/bash_history` file with this content?
```
ls -la
cd /home
git status
ls -la
git commit -m "Initial commit"
```

Then, you can add to the `tests` tables in TestParseHistory and TestHistoryFileNotFound to run the parser and check the bash parser implementation. 

I checked to make sure it parsed a bash history file 👍🏻

uncleBlobby committed:
Remove unnecessary logs from debugging.

Added history test file for bash parser, updated test commands to run for both parsers.

quentinlintz reviewed (changes requested):
Thank you! Last thing, sorry. Please check off the bash history support in the Roadmap README section.

uncleBlobby committed:
update readme roadmap
